infra/gcp/namespaces/namespace-user-role.yml

```diff
@@ -18,13 +18,13 @@ rules:
     resources: ["cronjobs", "jobs"]
     verbs: ["*"]
   - apiGroups: ["extensions"]
```

Comment:
We could delete this API group. @thockin @spiffxp WDYT ?

Reply:
Yeah, I think we should, once we've migrated everything away from it